README.md

```diff
@@ -45,20 +58,48 @@ Password: Dynamic33
 
 5. **Run the Analysis Code**
 
-In RStudio, open `vancouver-tree-height-geo.qmd` and run all code chunks.
+open a terminal (from the docker rstudio) and run the following commands:
+
+```
+Rscript scripts/00_download_data.R "https://opendata.vancouver.ca/api/explore\
+/v2.1/catalog/datasets/street-trees/exports/csv?lang=en&timezone=\
+America%2FLos_Angeles&use_labels=true&delimiter=%3B" "data/street-trees.csv"
+```
+```
+Rscript scripts/01_validate_data.R "data/street-trees.csv"
+```
+```
+Rscript scripts/02_eda.R "data/street-trees.csv" "results/figures/heatmap.png"\
+"results/tables/level_table.csv"
+```
+```
+Rscript scripts/03_stat_analysis.R "data/street-trees.csv"\
+"results/models/chi_squared_results.rds"
+```
+```
+quarto render report/vancouver-tree-height-geo.qmd --to html
+```
+
+### Clean up
 
-6. **Render the PDF**
+To shut down the container and clean up the resources, type `Cntrl` + `C` in the terminal where you launched the container, and then type `docker compose rm`
 
-Render the quarto document to a PDF using the `Render` button in RStudio.
+## Developer notes
 
-## Dependencies
-The following packages are required by the analysis, and all are installed in the `dynamic33` Docker image:
-- tidyverse
-- janitor
-- ggplot2
-- knitr
-- stringr
-- testthat
+### Adding a new dependency
+
+1. Add the dependency to the `Dockerfile` file on a new branch.
+
+2. Re-build the Docker image locally to ensure it builds and runs properly.
+
+3. Push the changes to GitHub. A new Docker
```

Comment:
I don't think we need manual line breaks like this. not sure tho. don't have a strong opinion

Reply:
don’t have a strong opinion either; I was just trying to avoid scroll boxes to the right. But I just looked at the 522 course notes, and they also have lots of scroll boxes. I think it should be fine, will delete those line breaks